Bottom: 3f28bbb547e83e35e9c1f5590b12818445f5ab1a
Top:    56ba609b0a9f7d6f7641b1856b84b56f2da2e1e4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 20:19:14 +0000

Disable perturbation attempt during intersection


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 3ef2e74..6f24bd1 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1158,6 +1158,7 @@ seg_in_seg_arc_line (struct info *i, struct seg *s1, struct seg *s2)
 
   printf ("  Intersect count is %i\n", cnt);
 
+#if 0
   if (cnt == 0)
     {
       printf ("Trying with a perturbation\n");
@@ -1172,6 +1173,10 @@ seg_in_seg_arc_line (struct info *i, struct seg *s1, struct seg *s2)
 
       printf ("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! <<<<<<\n");
     }
+#endif
+
+  if (cnt == 0)
+    return 0;
 
   if (cnt == 2)
     {
